documentation/modules/configuring/con-config-node-pool-roles.adoc

```diff
@@ -15,6 +15,8 @@ For example, you may have a node pool that contains nodes that perform dual brok
 In this case, you create a new node pool with nodes that act only as brokers, and then reassign partitions from the dual-role nodes to the new brokers.
 You can then switch the old node pool to a controller-only role.
 
+WARNING: Although it is possible to split node pool roles in this way, it not currently possible to transition a dedicated node pool containing only controller or broker nodes to a combined node pool where nodes operate as brokers and controllers.
```

Comment:
Works for me just just mentioning that transitions per se is possible but "semi-functional" , for example adding broker role would take a bit longer time and few failed reconciliations but desired nodes would roll  (in ~10 minutes).

Reply:
I don't think we want to mention in the docs that somehting is _semifunctonal_ because it timeouts after a long time 😉

Reply:
Right! That "mentioning" in my comment was "me mentioning it" not recommendation to mention it here 😀

Reply:
Yeah. No change on the warning. Thanks @jankalinic